spec/stat_tracker_spec.rb

```diff
@@ -20,6 +20,52 @@
     expect(@stat_tracker).to be_a(StatTracker)
   end
 
+  it "#highest_total_score" do
+    expect(@stat_tracker.highest_total_score).to eq 11
```

Comment:
Not sure if we can just pull from spec harness or if they want more descriptive tests? but either way it tests for the same thing